.github/workflows/ocwm-issue-collector.yml

```diff
@@ -54,14 +54,32 @@ jobs:
           console.log("workMeetings:" + JSON.stringify(workMeetings));
 
           for (let r = 0; r < repositories.length; r++) {
-            console.log(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${appendLabel}`);
-            const { data: items2add } = await mygithub.request(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${appendLabel}`, {
-            });
-            console.log("Issues to add:" + JSON.stringify(items2add));
 
 
-            const { data: discussions } = await mygithub.request(`GET /repos/${process.env.OWNER}/${repositories[r]}/discussions?labels=${appendLabel}`, {
-            });
+            const { data: items2add } = await mygithub.request(`GET /repos/${process.env.OWNER}/${repositories[r]}/issues?labels=${appendLabel}`);
+            console.log("Issues to add:" + JSON.stringify(items2add));
+
+            const query = `
+            query {
+              search(query: "repo:${process.env.OWNER}/${repositories[r]} is:open label:${appendLabel}", type: DISCUSSION, first: 5) {
+                edges {
+                  node {
+                    ... on Discussion {
+                      title
+                      body
+                      resourcePath
+                      author {
+                        login
+                      }
+                    }
+                  }
+                }
+              }
+            }`;  // Missing closing curly brace for the `try` block
```

Comment:
What's the purpose of this comment?

Reply:
Thanks for the review. That was a reminder that I forgot to remove it from the code but I'll remove. Thanks for the heads-up.